src/reconcile/rec_write.c

```diff
@@ -3345,6 +3354,91 @@ __rec_split_write_header(WT_SESSION_IMPL *session,
 }
 
 /*
+ * __rec_split_write_reuse --
+ *	Check if a previously written block can be reused.
+ */
+static bool
+__rec_split_write_reuse(WT_SESSION_IMPL *session,
+    WT_RECONCILE *r, WT_MULTI *multi, WT_ITEM *image, bool last_block)
+{
+	WT_PAGE_MODIFY *mod;
+	WT_MULTI *multi_match;
```

Comment:
in alphabetic order, WT_MULTI before WT_PAGE_MODIFY

Reply:
Thank you @sulabhM, fixed in 06634ac.